src/Shell/PackageNotFoundException.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CloudPatches\Shell;
+
+use Magento\CloudPatches\App\GenericException;
+
+/**
+ * Exception if a composer package could not be found for some reason (eg, symfony/process)
```

Comment:
At the risk of being pedantic, this could be written better as:

> Exception if a Composer package could not be found for some reason (eg, symfony/process).